src/Jackett.Common/Definitions/torrentmasters.yml

```
@@ -0,0 +1,151 @@
+---
+id: torrentmasters
+name: TorrentMasters
+description: "TorrentMasters is a Hungarian private tracker for MOVIES / TV / GENERAL"
```

```suggestion
description: "TorrentMasters is a HUNGARIAN private tracker for MOVIES / TV / GENERAL"
```
in line with current practices.